contracts/MANACrowdsale.sol

```diff
@@ -94,26 +99,21 @@ contract MANACrowdsale is ContinuousCrowdsale, WhitelistedCrowdsale, CappedCrowd
         TokenPurchase(msg.sender, beneficiary, weiAmount, tokens);
 
         forwardFunds();
-
-        return tokens;
     }
 
     function setWallet(address _wallet) onlyOwner public {
```

Comment:
Should `continuousSale`'s wallet be changed here as well? Otherwise there's no way to change it before calling `start` on it.